python/patched/astexport.py

```diff
@@ -102,21 +79,17 @@ def visit_field_Num_n(self, val):
             }
 
 
-def parse(source):
-    assert (isinstance(source, str))
-
-    atok = asttokens.ASTTokens(source, parse=True)
-
-    return atok
-
-
-def main():
-    source = "".join(fileinput.input())
-
-    tree = parse(source)
-    json = export_json(tree, True)
-    print(json)
+def export(atok):
+    exported_ast = DictExportVisitor(atok).visit(atok.tree)
 
+    exported_ast['comments'] = [
+        {
+            'ast_type': 'comment',
+            'value': token.string,
+            'start': token.startpos,
+            'end': token.endpos,
+        }
+        for token in atok.tokens if token.type == tokenize.COMMENT
```

Comment:
resolves https://github.com/prettier/prettier-python/pull/9/files#r158746577 and correspondingly fixes comment handling in python 2